dozer/cogs/levels.py

```diff
@@ -52,24 +53,25 @@ async def meesyncs(self, ctx):
         guild_id = ctx.guild.id
         progress_template = "Currently syncing from Mee6 API please wait... Page: {page}"
         msg = await ctx.send(progress_template.format(page="N/A"))
-        for page in range(0, 100000):
-            request = requests.get("https://mee6.xyz/api/plugins/levels/leaderboard/{guildID}?page={page}".format(guildID=guild_id, page=page))
-            data = request.json()
-            if len(data["players"]) > 0:
-                for user in data["players"]:
-                    ent = MemberXP(
-                        guild_id=int(guild_id),
-                        user_id=int(user["id"]),
-                        total_xp=int(user["xp"]),
-                        total_messages=int(user["message_count"]),
-                        last_given_at=ctx.message.created_at.replace(tzinfo=timezone.utc)
-                    )
-                    await ent.update_or_add()
-                if page % 2:
-                    await msg.edit(content=progress_template.format(page=page))
-            else:
-                break
-            time.sleep(1.25)  # Slow down api calls as to not anger cloudflare
+        for page in itertools.count():
+            async with aiohttp.ClientSession() as session:
```

Comment:
`ClientSession`s carry a lot of state and are meant to be reused for many requests. A single session should be created when the cog is created and reused between calls. For reference, see how [the `News` cog](https://github.com/FRCDiscord/Dozer/blob/b467ffd70d196aef48ec38c5ca9daddcd0c5fb2f/dozer/cogs/news.py) creates and uses its `http_session` attribute.